docs/accordion.md

```diff
@@ -1,131 +1,171 @@
 # Accordion
+
+A list of items that allows each item's content to be expanded and collapsed by clicking its title bar.
+
 ## Usage
 
 ```jsx
 import { Accordion } from 'radiance-ui';
+```
+
+<br>
+The accordion component expands to reveal hidden information. They should be used when you need to fit a large amount of content but don't want to visually overwhelm the user.
+
+<!-- STORY -->
+
+<!-- PROPS -->
+
+<br>
+
+### Standard Accordion
+
+The basic accordion for use on Curology's platform. Should be used whenever possible.
 
```

Comment:
I always tried to avoid making specific reference to Curology repos since this is Open Source. At the beginning  I even tried to avoid making examples with Emotion to make the library CSS framework agnostic but that was too far, at least review if we can remove the Curology reference

Reply:
That's my copy usage, I'm fine with removing it and just saying: 

"Basic accordion styling, contains standard border, shadow and padding."

Reply:
@daigof I appreciate your attention to detail on my PRs in this repo!